common/source/docs/common-zed.rst

```diff
@@ -27,9 +27,10 @@ Setup
 
 - connect the flight controller and TX1 as shown on the `NVidia TX1 as a Companion Computer <http://ardupilot.org/dev/docs/companion-computer-nvidia-tx1.html>`__ wiki page
 - flash the TX1 as described in the `Setup the TX1 <http://ardupilot.org/dev/docs/companion-computer-nvidia-tx1.html#setup-the-tx1>`__ section of the above wiki page but use the apsync-tx1-openkai image from `firmware.ardupilot.org <http://firmware.ap.ardupilot.org/Companion/apsync/beta/>`__
-- mount the ZED camera upside-down and using vibration dampeners to the front of the vehicle.  It can be mounted right-side-up after `connecting with ssh <http://ardupilot.org/dev/docs/apsync-intro.html#connecting-with-ssh>`__ and then modify `~/src/OpenKAI/kiss/apmCopter.kiss <https://github.com/yankailab/OpenKAI/blob/master/kiss/apmCopter.kiss#L60>`__ so "bFlip" is 1
-- in order to download the ZED camera's calibration parameters from stereolabs.com, connect the TX1 to the internet with an ethernet cable during powerup (this procedure is only required once)
-- disable APSync's video streaming (so that OpenKai can use the camera) by connecting your computer to APSync's access point (ssid=ardupilot, pw=ardupilot), then open a browser to `http://10.0.1.128:8000 <http://10.0.1.128:8000>`__ and click the red "Disable Auto Streaming" button (button should turn green) and reboot the TX1.
+- mount the ZED camera upside-down and using vibration dampeners to the front of the vehicle.  
+  It can be mounted right-side-up after `connecting with ssh <http://ardupilot.org/dev/docs/apsync-intro.html#connecting-with-ssh>`__ and then modify `~/src/OpenKAI/kiss/apmCopter.kiss <https://github.com/yankailab/OpenKAI/blob/master/kiss/apmCopter.kiss#L60>`__ so "bFlip" is `1`.
```

Comment:
Hi @yankailab - the apmCopter.kiss file no longer exists and the apCopter.kiss that (may) replace it is empty. So this instruction is now incorrect. Can you advise which file we would now edit?